README.md

```diff
@@ -37,25 +37,26 @@ Large Language Models (LLMs.) The "**lab**" in Instruct**Lab** 🐶 stands for
 
 ## ❓ What is `ilab`
 
-`ilab` is a Command-Line Interface (CLI) tool that allows you to:
+`ilab` is a Command-Line Interface (CLI) tool that allows you to perform the following actions:
 
 1. Download a pre-trained Large Language Model (LLM).
 1. Chat with the LLM.
 
-To add new knowledge and skills to the pre-trained LLM you have to add new information to the companion [taxonomy](https://github.com/instructlab/taxonomy.git) repository.
-After that is done, you can:
+To add new knowledge and skills to the pre-trained LLM, add information to the companion [taxonomy](https://github.com/instructlab/taxonomy.git) repository.
+
+After you have added knowledge and skills, you can perform the following actions:
 
 1. Use `ilab` to generate new synthetic training data based on the changes in your local `taxonomy` repository.
 1. Re-train the LLM with the new training data.
 1. Chat with the re-trained LLM to see the results.
 
-The full process is described graphically in the [workflow diagram](./docs/workflow.png).
+For an overview of the full workflow, see the [workflow diagram](./docs/workflow.png).
 
 > [!IMPORTANT]
-> It is important to understand that running InstructLab on a laptop will give you a low-fidelity approximation of both synthetic data generation (using the `ilab generate` command)
-> and model instruction tuning (using the `ilab train` command, which uses QLoRA.) The quality of the results you get using these tools on a laptop will not be as high-fidelity as they might be using
-> a larger teacher model and a different training method. We have optimized InstructLab to enable community members with modest hardware to be able to use the technique. If you have more sophisticated
-> hardware, you can configure InstructLab to use a larger teacher model [such as Mixtral](https://huggingface.co/docs/transformers/model_doc/mixtral) in order to achieve higher-fidelity results.
+> Running InstructLab on a laptop will provide a low-fidelity approximation of synthetic data generation (using the `ilab generate` command) and model instruction tuning (using the `ilab train` command,
+> which uses QLoRA.) The quality of the results will not be as high in fidelity as using a larger teacher model and a different training method. We have optimized InstructLab so that community members 
```

Comment:
nit: I don't remember seeing "high in fidelity" all that often in the wild.  Maybe turn around to avoid the negative, with something like:

> More sophisticated training methods and models are required to achieve higher quality.

or something like that?

Reply:
Hi @wking - thanks for the review!

I've submitted a new commit to incorporate that wording and shuffle around the sentences a little to convey the flow of ideas a little more cleanly.

What do you think?